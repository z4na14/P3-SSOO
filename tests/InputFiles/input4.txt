4 1 3 5 2 4 10 3 8 2
