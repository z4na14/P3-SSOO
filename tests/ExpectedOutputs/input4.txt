[OK][factory_manager] Process_manager with id 1 has been created.
[OK][factory_manager] Process_manager with id 2 has been created.
[OK][factory_manager] Process_manager with id 3 has been created.
[OK][process_manager] Process_manager with id 1 waiting to produce 5 elements.
[OK][process_manager] Process_manager with id 3 waiting to produce 2 elements.
[OK][process_manager] Process_manager with id 2 waiting to produce 10 elements.
[OK][process_manager] Belt with id 1 has been created with a maximum of 3 elements.
[OK][process_manager] Belt with id 3 has been created with a maximum of 8 elements.
[OK][queue] Introduced element with id 0 in belt 3.
[OK][queue] Introduced element with id 1 in belt 3.
[OK][queue] Introduced element with id 0 in belt 1.
[OK][queue] Introduced element with id 1 in belt 1.
[OK][queue] Introduced element with id 2 in belt 1.
[OK][queue] Obtained element with id 0 in belt 3.
[OK][queue] Obtained element with id 1 in belt 3.
[OK][process_manager] Process_manager with id 3 has produced 2 elements.
[OK][process_manager] Belt with id 2 has been created with a maximum of 4 elements.
[OK][queue] Introduced element with id 0 in belt 2.
[OK][queue] Introduced element with id 1 in belt 2.
[OK][queue] Introduced element with id 2 in belt 2.
[OK][queue] Introduced element with id 3 in belt 2.
[OK][queue] Obtained element with id 0 in belt 2.
[OK][queue] Obtained element with id 1 in belt 2.
[OK][queue] Obtained element with id 2 in belt 2.
[OK][queue] Obtained element with id 3 in belt 2.
[OK][queue] Obtained element with id 0 in belt 1.
[OK][queue] Introduced element with id 4 in belt 2.
[OK][queue] Obtained element with id 1 in belt 1.
[OK][queue] Obtained element with id 2 in belt 1.
[OK][queue] Introduced element with id 5 in belt 2.
[OK][queue] Introduced element with id 6 in belt 2.
[OK][queue] Introduced element with id 7 in belt 2.
[OK][queue] Introduced element with id 3 in belt 1.
[OK][queue] Introduced element with id 4 in belt 1.
[OK][queue] Obtained element with id 3 in belt 1.
[OK][queue] Obtained element with id 4 in belt 1.
[OK][process_manager] Process_manager with id 1 has produced 5 elements.
[OK][queue] Obtained element with id 4 in belt 2.
[OK][queue] Obtained element with id 5 in belt 2.
[OK][queue] Obtained element with id 6 in belt 2.
[OK][queue] Obtained element with id 7 in belt 2.
[OK][queue] Introduced element with id 8 in belt 2.
[OK][factory_manager] Process_manager with id 1 has finished.
[OK][queue] Introduced element with id 9 in belt 2.
[OK][queue] Obtained element with id 8 in belt 2.
[OK][queue] Obtained element with id 9 in belt 2.
[OK][process_manager] Process_manager with id 2 has produced 10 elements.
[OK][factory_manager] Process_manager with id 2 has finished.
[OK][factory_manager] Process_manager with id 3 has finished.
[OK][factory_manager] Finishing.
